### DISCUSSION TITLE
DeSo Public Key Update

### DISCUSSION THREAD

**Discussion by 54rrdeaniel** [deso-protocol/dips#159] [Medium CIPs]
The BitClout protocol has recently been renamed DeSo. I think the BC at the beginning of the Public key should change. I suggest 'SO' instead of 'BC'.

BitClout Protocol
BC1YLgmabmhDTrz2cnaAj3fM62KhnJbxTSRKJUnBUbL1speV621zj45

DeSo Protocol
SO1YLgmabmhDTrz2cnaAj3fM62KhnJbxTSRKJUnBUbL1speV621zj45

**Comment by tijno** [comment]
What are the benefits of doing this and how would transition be managed?

How would you address problem of user confusion & errors due to using old format with upgraded nodes or new format with non-upgraded nodes ? 

Is any other coin using the SO prefix?

**Comment by ItsAditya-xyz** [comment]
I think there are many many such things that cannot be changed (and i don't think there is any point to change it). One being the prefix of public keys. If you see the core code, API responses, you will see terms like `BitCloutNanos`, `BitCloutLockedNanos` etc. I don't think this needs to be changed as this will hurt already built projects.

**Reply by stet** [reply]
yes, which is why when nader said the blockchain never had a name, he is not being honest. he just realized what i was saying from beginning... that bitclout was a terrible name for the entire stack (but ok for a proof-of-concept "product"). he has made it sound like he finally got around to naming the protocol/blockchain but i don't believe he realized it was necessary early on.

as for the change requested here, i will say its too late and just deal with it until the entire software is rebuilt (which i think is likely to happen).
